### PR TITLE
Respond to reported feedback

### LINKED ISSUE #6582: Feedback on String Interpolation
I had a friend run through all of this last night.  He said steps 4-7 for the String Interpolation were explained in a way that confused him, he found it hard to understand.

He also mentioned when introducing the "Trim" method, that the example seemed over complicated, and that parentheses character was introduced, but then not explained.  So he was unsure what it does.  

He suggested using a simpler or less-complicated example for Trim.

The said the rest of the entire tutorial was a pleasure, easy to follow and understand.  :)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: df2cc16c-fcf9-41e6-a1e5-92dfc2869495
* Version Independent ID: 4421a02c-f819-1483-12c6-7dab4f289a46
* Content: [C# interactive tutorials - C# quickstarts](https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/#feedback)
* Content Source: [docs/csharp/quick-starts/index.md](https://github.com/dotnet/docs/blob/master/docs/csharp/quick-starts/index.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
A customer reported that the `hello world` quick start was confusing in spots.

I re-worked the descriptions in areas that were reported as confusing.

I added text to encourage readers to type the code into the window and explore intellisense.

Fixes #6582 

/cc @Shehackspurple

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?branch=pr-en-us-7771)

### CODE REVIEW

**Comment by BillWagner** [comment]
cc @LadyNaggaga I'm referencing the new intellisense feature in this PR. Pointing readers to this experience is powerful!

File: docs/csharp/quick-starts/hello-world.yml [hunk 1]
```diff
@@ -146,7 +151,7 @@ items:
...
 
-    There are other methods available to work with a string. For example, you've probably used a search and replace command in an editor or word processor. The <xref:System.String.Replace%2A> method does something similar in a `string`. It searches for a substring, and replaces it with different text. The <xref:System.String.Replace%2A> method takes two **parameters**. These are the strings between the parentheses. The first string is the text to search for. The second string is the text to replace it with. Try it for yourself. Add this code:
+    There are other methods available to work with a string. For example, you've probably used a search and replace command in an editor or word processor. The <xref:System.String.Replace%2A> method does something similar in a `string`. It searches for a substring, and replaces it with different text. The <xref:System.String.Replace%2A> method takes two **parameters**. These are the strings between the parentheses. The first string is the text to search for. The second string is the text to replace it with. Try it for yourself. Add this code. Type it in to see the hints as you start typing `.Re` after the `sayHello` variable:
```
Comment (rpetrusha) [inline docs/csharp/quick-starts/hello-world.yml]: no comma after substring

File: docs/csharp/quick-starts/hello-world.yml [hunk 2]
```diff
@@ -128,7 +132,8 @@ items:
   content: |
     You've been using a **method**, <xref:System.Console.WriteLine%2A?displayProperty=nameWithType>, to print messages. A **method** is a block of code that implements some action. It has a name, so you can access it. 
     
-    The `string` type has many useful methods. Let's explore the <xref:System.String.Trim%2A> method, and related methods <xref:System.String.TrimStart%2A> and <xref:System.String.TrimEnd%2A>. These methods remove leading and trailing spaces:
+    Suppose your strings may have leading or trailing spaces that you don't want to display. You want to **trim** the spaces from the strings.
+    The <xref:System.String.Trim%2A> method, and related methods <xref:System.String.TrimStart%2A> and <xref:System.String.TrimEnd%2A> do that work. You can just use use those methods to remove leading and trailing spaces. Try the following code:
```
Comment (rpetrusha) [inline docs/csharp/quick-starts/hello-world.yml]: no comma after method

File: docs/csharp/quick-starts/hello-world.yml [hunk 3]
```diff
@@ -128,7 +132,8 @@ items:
   content: |
     You've been using a **method**, <xref:System.Console.WriteLine%2A?displayProperty=nameWithType>, to print messages. A **method** is a block of code that implements some action. It has a name, so you can access it. 
     
-    The `string` type has many useful methods. Let's explore the <xref:System.String.Trim%2A> method, and related methods <xref:System.String.TrimStart%2A> and <xref:System.String.TrimEnd%2A>. These methods remove leading and trailing spaces:
+    Suppose your strings may have leading or trailing spaces that you don't want to display. You want to **trim** the spaces from the strings.
```
Comment (rpetrusha) [inline docs/csharp/quick-starts/hello-world.yml]: nit: may have --> have
no comma after method

**Review by rpetrusha — APPROVED** [review]
Left a couple of nits, @BillWagner, but your changes look really good. You can merge when you're ready.

File: docs/csharp/quick-starts/hello-world.yml [hunk 4]
```diff
@@ -80,16 +80,19 @@ items:
...
-    If you add a `$` before the opening quote of the string, you can then include expressions, like the `aFriend` variable, inside the string in curly brackets.
+    If you add a `$` before the opening quote of the string, you can then include variables, like `aFriend`, inside the string between curly braces. Give it a try:
 
 
     ```csharp
     Console.WriteLine($"Hello {aFriend}");
     ```
 
-    Click **Run** again to see the results.
+    Click **Run** again to see the results. Instead of "Hello {aFriend}", the message should be "Hello Maira".
```
Comment (mairaw) [inline docs/csharp/quick-starts/hello-world.yml]: I haven't noticed my name got added to the docs before 😄

**Review by mairaw — APPROVED** [review]
Looks great. Left a few comments.

**Pull request merged**